app/build.gradle

```diff
@@ -4,35 +4,33 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-kapt'
 
-apply plugin: 'kotlin-android-extensions'
-
```

Comment:
why you remove this?

Reply:
I think that `Kotlin extensions` are deprecated and replaced with `ViewBinding`. @hrodrick I'm right?

Reply:
Kotlin extension are deprecated. We should use view binding and @Parcelize. Good update!

Reply:
Yes, Kotlin extensions are deprecated and kotlin synthetics is replaced by View & Data binding